component-interaction/src/app/sort.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { UserModel } from "./user.model";
+
+@Pipe({
+	name: "sort"
+})
+
+export class SortPipe implements PipeTransform {
+	transform(users: UserModel[], order?: string): UserModel[] {

```

Comment:
```suggestion
	transform(users: UserModel[], order?: OrderTypeEnum): UserModel[] {

```
define an enum for better readability